src/events/deposits.upsert.service.ts

```diff
@@ -255,4 +257,37 @@ export class DepositsUpsertService {
       );
     });
   }
+
+  async syncDepositedIron(): Promise<void> {
+    const aggregate = await this.prisma.deposit.aggregate({
+      _sum: {
+        amount: true,
+      },
+    });
```

Comment:
This requires dependency injection of `influxDbService`, but I don't know where to add it. There is no corresponding `*.module.ts`

Note that I see this error when I run a unittest locally.